src/components/Toast/index.js

```diff
@@ -35,14 +36,17 @@ export default class Button extends PureComponent {
     kind: 'default',
   }
 
-  toastRef = React.createRef()
+  containerRef = React.createRef()
 
   state = {
     show: false,
   }
 
   show = () => {
-    this.setState({ show: true })
+    this.setState({
+      height: this.containerRef.current.offsetHeight,
```

Comment:
Setting the height here to ensure that the styles applied to the page already exist (was having issues with dynamically loaded styles, and the height calc being without any CSS)